kyo-core/shared/src/main/scala/kyo/Queue.scala

```diff
@@ -257,7 +257,6 @@ object Queue:
         object Unsafe:
             extension [A](self: Unsafe[A])
                 def add(value: A)(using AllowUnsafe, Frame): Unit = discard(self.offer(value))
-                def safe: Unbounded[A]                            = self
```

Comment:
the compiler started warning for extension methods that can never be selected. In this case, `Unsafe` already has a `safe` method